DependencyInjection/SonataMediaExtension.php

```diff
@@ -252,7 +256,7 @@ public function registerDoctrineMapping(array $config)
             ),
         ));
 
-        if (interface_exists('Sonata\ClassificationBundle\Model\CategoryInterface')) {
+        if (false === $config['force_disable_category'] && null !== $config['class']['category'] && false !== $config['class']['category']) {
```

Comment:
Can you show me the line, where we could set `$config['class']['category']` to false?